src/main/java/rosenfeld/boggle/Dictionary.java

```diff
@@ -4,22 +4,27 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
 public class Dictionary {
 
     private final Map<String, String> wordsToDefinitions = new HashMap<>();
+    private final List<String> wordList;
```

Comment:
You may as well edit Dictionary to only have the wordList and get rid of the Map since you're not using it.